app/src/main/java/io/gnosis/safe/ui/splash/SplashViewModel.kt

```diff
@@ -40,6 +42,14 @@ constructor(
         tracker.setNumSafes(numSafes)
         tracker.setNumKeysImported(credentialsRepository.ownerCount(Owner.Type.IMPORTED))
         tracker.setNumKeysGenerated(credentialsRepository.ownerCount(Owner.Type.GENERATED))
+
+        kotlin.runCatching {
+            val chainInfos = chainInfoRepository.getChainInfo()
```

Comment:
Why move this to splash?

Reply:
we have to load chain details only on app start and not every time app comes back from background. Also having this in view model makes it easier to verify and test